packages/node/src/ethereum/setup-commitment.ts

```diff
@@ -60,7 +61,7 @@ export class SetupCommitment extends MultisigCommitment {
 
   private getSaltForDependencyNonce() {
     return keccak256(
-      defaultAbiCoder.encode(["uint256"], [DependencyValue.NOT_UNINSTALLED])
+      defaultAbiCoder.encode(["uint256"], [DependencyValue.NOT_CANCELLED])
```

Comment:
What does `CANCELLED` mean exactly? I feel like `UNINSTALLED` has a specific meaning.

Maybe `NOT_INVALIDATED` is more precise?

Reply:
The commitment is `cancelled` when the app is `uninstalled`